store/sqlstore/channel_store.go

```diff
@@ -1302,3 +1302,23 @@ func (s SqlChannelStore) GetMembersByIds(channelId string, userIds []string) sto
 		}
 	})
 }
+
+func (s SqlChannelStore) UpdateActiveByUser(userId string, time int64) store.StoreChannel {
```

Comment:
Not sure how this works with caching and the clients?

When a channel gets reactivated how do the connected clients and channel caches know?

Reply:
I'm going to add a websocket event for clients. I guess I also need to add some cache invalidation.